Mage.Sets/src/mage/cards/c/ClavilenoFirstOfTheBlessed.java

```diff
@@ -0,0 +1,122 @@
+package mage.cards.c;
+
+import java.util.UUID;
+
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.AttacksWithCreaturesTriggeredAbility;
+import mage.abilities.common.DiesSourceTriggeredAbility;
+import mage.abilities.effects.ContinuousEffect;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.CreateTokenEffect;
+import mage.abilities.effects.common.DrawCardSourceControllerEffect;
+import mage.abilities.effects.common.continuous.AddCardSubTypeTargetEffect;
+import mage.abilities.effects.common.continuous.GainAbilityTargetEffect;
+import mage.abilities.keyword.FlyingAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.constants.Outcome;
+import mage.constants.SubType;
+import mage.constants.SuperType;
+import mage.filter.FilterPermanent;
+import mage.filter.common.FilterAttackingCreature;
+import mage.filter.predicate.Predicates;
+import mage.game.Game;
+import mage.game.permanent.token.TokenImpl;
+import mage.target.TargetPermanent;
+
+public class ClavilenoFirstOfTheBlessed extends CardImpl {
+
+    private static final FilterPermanent filter
+            = new FilterAttackingCreature("attacking Vampire that isn't a Demon");
+
+    static {
+        filter.add(Predicates.and(
```

Comment:
(fyi it isn't necessary to wrap in `Predicates.and()`, you can just have two `filter.add()` calls)